src/app/sign-in/components/form-sign-in/form-sign-in.component.ts

```diff
@@ -254,7 +254,8 @@ export class FormSignInComponent implements OnInit, AfterViewInit {
       .subscribe((requestInfoForm: RequestInfoForm) => {
         if (
           requestInfoForm.error === 'invalid_grant' ||
-          requestInfoForm.error === 'invalid_scope'
+          requestInfoForm.error === 'invalid_client' ||
```

Comment:
Since all the requestInfoForm errors are been handle on the service, and those produce a redirect with the query parameters. 

I believe we don't need to handle these errors again on this component. Please double check

Reply:
I will remove the other ones but I would leave the invalid_grant error because this one has to send information to his parent component and display the correct error message